Lib/test/support/__init__.py

```diff
@@ -1887,13 +1893,12 @@ def restore(self):
 
 
 def with_pymalloc():
-    import _testcapi
-    return _testcapi.WITH_PYMALLOC and not Py_GIL_DISABLED
+    WITH_PYMALLOC = bool(sysconfig.get_config_var("WITH_PYMALLOC"))
```

Comment:
Apparently, this does not work well on Windows. How does `get_config_var(s)` relate to `PC/pyconfig.h.in`?